### PR TITLE
Generate the page ASCII output code

### PR DESCRIPTION
**Pull request by adbancroft**
Parse the INI file and generate `printPageAscii()`.

Parsing code is:

- Python (cross-platform, widely used)
- Integrated into the PIO build pipeline

We will need to keep the code & INI file more closely synchronized. But that's a good thing.

### CODE REVIEW

File: reference/speeduino.ini [hunk 1]
```diff
@@ -275,8 +275,8 @@ page = 1
 
       reqFuel       = scalar, U08,      24,        "ms",        0.1,       0.0,   0.0,    25.5,      1
       divider       = scalar, U08,      25,        "",          1.0,       0.0
-      alternate     = bits,   U08,      26, [0:0], "Simultaneous", "Alternating"
-      multiplyMAP1   = bits,   U08,      26, [1:1], "No",       "Yes"
+      injTiming     = bits,   U08,      26, [0:0], "Simultaneous", "Alternating"
```
Comment (noisymime) [inline reference/speeduino.ini:278]: Unfortunately `alternate` is a reserved variable name used by TunerStudio. Changing this breaks the `req_fuel` calculator and so will need to be left the same.

File: reference/speeduino.ini [hunk 2]
```diff
@@ -286,7 +286,7 @@ page = 1
 
       ; Config1
       mapSample     = bits,   U08,      36, [0:1], "Instantaneous", "Cycle Average", "Cycle Minimum", "Event Average"
-      twoStroke     = bits,   U08,      36, [2:2], "Four-stroke", "Two-stroke"
+      strokes     = bits,   U08,      36, [2:2], "Four-stroke", "Two-stroke"
```
Comment (noisymime) [inline reference/speeduino.ini:289]: Same as with `alternate` above. This is a reserved variable name and must be kept (unfortunately)

**Comment by adbancroft** [comment]
@noisymime Is there a list of the TS reserved words somewhere?

Hilariously, TS didn't give me any errors/warnings :-(

File: reference/speeduino.ini (hunk 1 above)
Comment (adbancroft) [reply reference/speeduino.ini]: Fixed

File: reference/speeduino.ini (hunk 2 above)
Comment (adbancroft) [reply reference/speeduino.ini]: Fixed

**Comment by adbancroft** [comment]
Reworked this PR to improve back compat & reduce churn: no name changes to INI entry names or codebase

**Pull request closed without merging**